.github/workflows/test.yaml

```diff
@@ -2,9 +2,6 @@ name: Test
 
 on:
   pull_request:
-  push:
-    branches-ignore:
-      - main  # pushes to main are handled by deploy.yaml
```

Comment:
**question** The comment here makes it sound like it prevented a double run on push already, from both `test`, and `deploy` (which runs `test` as well). What changed to make it unnecessary?

Reply:
Nothing changed, it just got a bit annoying

Every time you push to a PR branch it runs all the checks for both `pull_request` and `push` 😣

The comment shows we only de-duplicated `main` branch in the past

Reply:
See also:

* https://github.com/alphagov/govuk-frontend/commit/4690934fac627eedeb79208569719fb99a4953a8
* https://github.com/alphagov/govuk-design-system/commit/dc6b2d64db5e6584d1e8a9c09391650da2b4ed31